### PR TITLE
Init password gen for links

### PR DESCRIPTION
**Pull request by JanAckermann**
## Related Issue

- Fixes https://github.com/owncloud/web/issues/9666

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by JammingBen — COMMENTED** [review]
Minor typing improvement from my side, rest LGTM.

File: packages/design-system/src/components/OcModal/OcModal.vue [hunk 1]
```diff
@@ -371,6 +372,14 @@ export default defineComponent({
       required: false,
       default: () => ({})
     },
+    /**
+     * Method to generate random password for the input
+     */
+    inputGeneratePasswordMethod: {
+      type: Function,
```
[the remark this answers is not in the document]
Comment (JammingBen) [reply packages/design-system/src/components/OcModal/OcModal.vue]: Ah it's actually `...args: unknown[]`

[the remark this answers is not in the document]
Comment (JanAckermann) [reply packages/design-system/src/components/OcModal/OcModal.vue]: ihmo super hard to read, but ok...

[the remark this answers is not in the document]
Comment (JammingBen) [reply packages/design-system/src/components/OcModal/OcModal.vue]: Still better than no type 😄 The string return type is the important thing here. `...args: unknown[]` simply makes no assumption about the amount and type of arguments here.

**Pull request merged**